pages/15.Reports/docs.en.md

```diff
@@ -34,6 +34,16 @@ Some reports have graphs available. Select the graph desired from the left list
 
 ## Dashboard Widget
 
-Each graph of each report is made available as a widget on the dashboard allowing complete customization of the dashboard. 
- 
-![](widget.png)
\ No newline at end of file
+Each graph of each report is made available as a widget on the dashboard allowing complete customization of the dashboard.
+
+![](widget.png)
+
+## Schedule
+
+Enable or disable sending reports via emails by using the toggle switch. You can schedule emails to send reports to one or more email address. In the To field, enter a comma-separated list of email addresses and set the frequency of sending reports by choosing day, week, or month from the drop-down list. To be able to schedule emails to send reports, run the command:
+
+```
+php /path/to/mautic/app/console mautic:reports:scheduler [--report=ID]
+```
+
```

Comment:
You may wish to add a link here to the cron jobs page where they can get more information (using a link macro)

Reply:
Added a macro link to the cron jobs page.

Reply:
I would recommend using something like `[cron-jobs]` for the name of the macro so that it is clear to someone reviewing this later where to find it being used on the page